packages/cli/generation/local-generation/local-workspace-runner/src/runLocalGenerationForWorkspace.ts

```diff
@@ -119,7 +125,22 @@ async function writeFilesToDiskAndRunGenerator({
     if (firstLocalOutputItem == null) {
         return;
     }
-    await rm(absolutePathToLocalOutput, { force: true, recursive: true });
+    // await rm(absolutePathToLocalOutput, { force: true, recursive: true });
+    const dotFernIgnoreFile = path.join(absolutePathToLocalOutput, ".fernignore");
+    let filesToBeIgnoredByFern: string[] = [".fernignore"];
+    const dotFernIgnoreFileExists = existsSync(dotFernIgnoreFile);
```

Comment:
there should be a utility function here so that you can check if file exists called `doesPathExist`

https://github.com/fern-api/fern/blob/c91a4ba9028abd704d6de698510507d751e1ba7b/packages/commons/fs-utils/src/doesPathExist.ts#L4

Reply:
Oh! I missed that. I’ll make the adjustments